conf.yaml

```diff
@@ -1253,6 +1274,7 @@ contents:
                 index:      docs/guide/index.asciidoc
                 current:    7.4
                 branches:   [ master, 7.5, 7.4, 7.3, 7.2, 7.1, 7.0, 6.8, 6.7, 6.6, 6.5, 6.4, 6.3, 6.2, 6.1, 6.0 ]
+                live:       *stacklive
```

Comment:
APM doesn't build `7.x` documentation. Will using `*stacklive` here (which lists `7.x` as a live branch) cause a problem?

Reply:
It *ought* to be fine if the `live` list is "bigger" than the branches list.